.circleci/config.yml

```diff
@@ -20,48 +20,18 @@ executors:
           - cluster.name: elasticsearch
           - action.auto_create_index: true
 jobs:
-  build:
-    executor: libraries
-    steps:
-      - checkout
-      - restore_cache:
-          keys:
-            # when lock file changes, use increasingly general patterns to restore cache
-            - v1-gem-cache-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
-            - v1-gem-cache-{{ arch }}-{{ .Branch }}-
-            - v1-gem-cache-{{ arch }}-
-      - run: sudo apt-get -y -qq update
-      - run: sudo apt-get -y --no-install-recommends install cmake libpq-dev
-      - ruby/bundle-install
-      - save_cache:
-          key: v1-gem-cache-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
-          paths:
-            - vendor/bundle
```

Comment:
Couldn't figure out a reason why we'd need the `build` section, when it's almost identical to the test step.